aiven/resource_project.go

```diff
@@ -516,11 +517,15 @@ func setProjectTerraformProperties(d *schema.ResourceData, client *aiven.Client,
 	if err := d.Set("billing_address", project.BillingAddress); err != nil {
 		return diag.FromErr(err)
 	}
-	if err := contactEmailListForTerraform(d, "billing_emails", project.BillingEmails); err != nil {
-		return diag.FromErr(err)
+	if _, ok := d.GetOk("billing_emails"); ok {
+		if err := contactEmailListForTerraform(d, "billing_emails", project.BillingEmails); err != nil {
+			return diag.FromErr(err)
+		}
 	}
-	if err := d.Set("country_code", project.CountryCode); err != nil {
-		return diag.FromErr(err)
+	if _, ok := d.GetOk("country_code"); ok {
+		if err := d.Set("country_code", project.CountryCode); err != nil {
+			return diag.FromErr(err)
+		}
 	}
```

Comment:
This `GetOk` / `Set` pattern probably can be extracted to a function

Reply:
done